### PR TITLE
Fix y-axis formatting and spacing

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard4**
## Summary

Small Y-axis tweaks:
* make spacing bitter for larger numbers to fit them better
* default number formatting

### CODE REVIEW

File: packages/app/src/components-styled/line-chart/index.tsx [hunk 1]
```diff
@@ -130,11 +124,22 @@ export function LineChart<T extends Value>({
...
-  const yDomain = useMemo(() => [0, calculateYMax(trendsList, signaalwaarde)], [
+  const yMax = useMemo(() => calculateYMax(trendsList, signaalwaarde), [
     trendsList,
     signaalwaarde,
   ]);
 
+  const yDomain = useMemo(() => [0, yMax], [yMax]);
+
+  // Increase space for larger labels
+  padding.left = Math.max(yMax.toFixed(0).length * 10, defaultPadding.left);
```
Comment (VWSCoronaDashboard10) [inline packages/app/src/components-styled/line-chart/index.tsx:135]: please don't mutate, this is potentially very buggy. Instead, clone the padding object and apply the new padding on that instance, eg:
```tsx
padding = { ...padding, left: ... }

**Review by VWSCoronaDashboard10 — APPROVED** [review]
Approved, but please have a look at the mutation thingy :-)

File: packages/app/src/components-styled/line-chart/index.tsx (hunk 1 above)
Comment (VWSCoronaDashboard1) [reply packages/app/src/components-styled/line-chart/index.tsx:135]: I'm not sure whether we should try to adjust the left padding automatically. Isn't it just an option to adjust the margin manually for a specific instance of the chart?
I don't think throughout the lifetime of the data there will be fluctuations between 10 and 10.000.000 in the given data, so a hardcoded solution would probably be just as good.
Just a thought...

**Review by VWSCoronaDashboard1 — APPROVED** [review]

File: packages/app/src/components-styled/line-chart/index.tsx (hunk 1 above)
Comment (VWSCoronaDashboard4) [reply packages/app/src/components-styled/line-chart/index.tsx:135]: good catch!

**Pull request merged**